### PR TITLE
FEATURE: make outputResults parameter configurable

### PR DESCRIPTION
**Pull request by das-nagnag**

### CODE REVIEW

**Comment by das-nagnag** [comment]
Thank you for the feedback! That's a very good call, I actually didn't even think of that, my bad 😅 I'll try to implement that shortly

File: README.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ The following parameters are supported by all queues:
 | className               | string  |       - | FQN of the class implementing the queue                                                                                         |
 | maximumNumberOfReleases | integer |       3 | Max. number of times a message is re-<br>released to the queue if a job failed                                                  |
 | executeIsolated         | boolean |   FALSE | If TRUE jobs for this queue are executed in a separate Thread. This makes sense in order to avoid memory leaks and side-effects |
+| outputResults           | boolean |   FALSE | If set to true, forwards the full output (stdout + stderr) of the respective job to the stdout of its "parent"                  |
```
Comment (bwaidelich) [inline README.md:192]:
```suggestion
| outputResults           | boolean |   FALSE | If TRUE, the full output (stdout + stderr) of the respective job is forwarded to the stdout of its "parent" (only applicable if `executeIsolated` is `true`)  |
```

**Review by bwaidelich — COMMENTED** [review]
you know that you can collect multiple changes into one commit, right? :D 

Looking great, just a mini suggestion regarding the description of the new setting

File: README.md (hunk 1 above)
Comment (das-nagnag) [reply README.md]: good point, thanks!

**Comment by das-nagnag** [comment]
> > [das-nagnag](https://github.com/das-nagnag) added 5 commits [16 minutes ago](https://github.com/Flowpack/jobqueue-common/pull/65#commits-pushed-05bc416)
> 
> you know that you can collect multiple changes into one commit, right? :D
> 
> Looking great, just a mini suggestion regarding the description of the new setting

Yeah, sorry for all the tiny commits. My phpStorm decided today would be the perfect day to stop working properly and since I didn't have the time or energy to get it up and running again yet, I just made all the changes through githubs online editor – which is both painful and slow :D

**Pull request merged**